### ISSUE TITLE
Markup Line Glyph orientations

### ISSUE THREAD

**Issue by torquil** [Slicer/Slicer#4908]
In the latest 3DSlicer nightly on Linux, I'm creating some Line markups in one of the slice windows (red in this case). I noticed that the glyphs on the ends of the line are oriented independently of the orientation of the line. E.g. the Dash2D, ThickArray2D and Arrow2D (and so on) seem to always point horizontally. (Perhaps there is an option that I have not noticed, though....).

When using the Line markup to indicate a distance, it would be nice to have the Dash2D endpoint glyph at a normal angle to the line. So an option to specify the glyph orientation relative to the line orientation in the slice view would be useful, or at least to lock it to be normal to the line. (with generalizations to triange, square and diamond glyphs).

**Comment by pieper** [comment]
Yes, this is a byproduct of using the 2D glyphs.  Agreed it would be nice to have other options.
